rbc/tests/test_omnisci_math.py

```diff
@@ -0,0 +1,418 @@
+
+        if fn_name in ['_logaddexp', '_logaddexp2']:
+            descr, result = omnisci.sql_execute(
+                'select x, y, {fn_name}(x, y) from {omnisci.table_name}'
+                .format(**locals()))
+
+            for x, y, v in list(result):
+                assert(np.isclose(np_fn(x, y), v))
+        else:
+            descr, result = omnisci.sql_execute(
+                'select x, {fn_name}(x) from {omnisci.table_name}'
+                .format(**locals()))
+
+            for x, v in list(result):
+                assert(np.isclose(np_fn(x), v))
+
+
+def test_rational_funcs(omnisci):
+    omnisci.reset()
+
+    @omnisci('int(int, int)')  # noqa: F811
+    def lcm(i, j):
+        return np.lcm(i, j)
+
+    @omnisci('int(int, int)')  # noqa: F811
+    def gcd(i, j):
+        return np.gcd(i, j)
+
+    omnisci.register()
+
+    for fn_name in ['lcm', 'gcd']:
+        np_fn = getattr(np, fn_name)
+
+        descr, result = omnisci.sql_execute(
+            'select i, j, {fn_name}(i, j) from {omnisci.table_name}'
+            .format(**locals()))
+
+        for i, j, v in list(result):
+            assert(np.isclose(np_fn(i, j), v))
+
+
+def test_spd_funcs(omnisci):
+    omnisci.reset()
+
+    @omnisci('i32(i32[])')
+    def _sum(x):
+        return np.sum(x)
+
+    @omnisci('i64(i32[])')
+    def _prod(x):
+        return np.prod(x)
+
+    omnisci.register()
+
+    for fn_name in ['_sum', '_prod']:
+        np_fn = getattr(np, fn_name.lstrip('_'))
+
+        descr, result = omnisci.sql_execute(
+            'select t, {fn_name}(t) from {omnisci.table_name}'
+            .format(**locals()))
+
+        for t, v in list(result):
+            assert(np.isclose(np_fn(t), v))
+
+
+def test_arithmetic_funcs(omnisci):
+    omnisci.reset()
+
+    @omnisci('double(double, double)')  # noqa: F811
+    def add(i, j):
+        return np.add(i, j)
+
+    @omnisci('double(double)')  # noqa: F811
+    def reciprocal(x):
+        return np.reciprocal(x)
+
+    @omnisci('double(double)')  # noqa: F811
+    def negative(x):
+        return np.negative(x)
+
+    @omnisci('double(double, double)')  # noqa: F811
+    def multiply(x, y):
+        return np.multiply(x, y)
+
+    @omnisci('double(double, double)')  # noqa: F811
+    def divide(x, y):
+        return np.divide(x, y)
+
+    @omnisci('double(double, double)')  # noqa: F811
+    def _power(x, y):
+        return np.power(x, y)
+
+    @omnisci('double(double, double)')  # noqa: F811
+    def subtract(x, y):
+        return np.subtract(x, y)
+
+    @omnisci('double(double, double)')  # noqa: F811
+    def true_divide(x, y):
+        return np.true_divide(x, y)
+
+    @omnisci('double(double, double)')  # noqa: F811
+    def floor_divide(x, y):
+        return np.floor_divide(x, y)
+
+    @omnisci('double(double, double)')
+    def _fmod(x, y):
+        return np.fmod(x, y)
+
+    @omnisci('int(int, int)')  # noqa: F811
+    def mod(x, y):
+        return np.mod(x, y)
```

Comment:
`np.mod` works also for double inputs. So, apply also `@omnisci('double(double, double)')` to `mod`?